pairinteraction/SystemTwo.cpp

```diff
@@ -152,6 +152,17 @@ void SystemTwo::setConservedMomentaUnderRotation(const std::set<int> &momenta) {
     }
 }
 
+void SystemTwo::setOneAtomBasisvectors(const std::vector<std::array<size_t, 2>> &indices) {
+    // Check that all indices are unique
+    std::vector<std::array<size_t, 2>> tmp(indices);
+    std::sort(tmp.begin(), tmp.end());
+    if (std::adjacent_find(tmp.begin(), tmp.end()) != tmp.end()) {
+        throw std::runtime_error("The indices are not unique.");
+    }
```

Comment:
Might be nice to say which indices are duplicate:
```suggestion
    if (auto const it = std::adjacent_find(tmp.begin(), tmp.end()); it != tmp.end()) {
        std::string const which = "{" + std::to_string((*it)[0]) + "," + std::to_string((*it)[1]) + "}";
        throw std::runtime_error("The indices are not unique: " + which);
    }
```